kernel/Dockerfile

```diff
@@ -98,8 +98,16 @@ RUN make defconfig && \
     cp System.map /out && \
     ([ -n "${DEBUG}" ] && cp vmlinux /out || true)
 
+# WireGuard
+RUN TAG="0.0.$(curl -sSL https://git.zx2c4.com/WireGuard/info/refs/ | sed -n 's/.*refs\/tags\/0\.0\.\([0-9]\+\)$/\1/p' | sort -nr | head -n 1)" && \
```

Comment:
In this line here, I automatically determine the latest version by examining the tags. I like this, but I would understand if you prefer to instead hard code a particular snapshot version instead. Either way is fine with me; just let me know.